ffs/minerselector/sr2/sr2_test.go

```diff
@@ -0,0 +1,55 @@
+package sr2
```

Comment:
These are two tests that are skipped by default. They're useful as wired tests when connecting to an external Lotus node. It's useful to have them handy to tests things in testnet since it's impossible to test this selector in a devnet environment.